src/website/views/components/accordion/_examples.pug

```diff
@@ -77,7 +77,7 @@ h3 Base
         </div>
       </div>
       <div class="chi-accordion">
-        <div class="chi-accordion__header">
+        <div class="chi-accordion__trigger">
           <i class="chi-icon icon-chevron-down"></i>
           <div class="chi-accordion__title">Accordion 4</div>
         </div>
```

Comment:
@janibolkvadze missing the js script that initializes the accordion in the copy section